doc/api/errors.md

```diff
@@ -1713,12 +1729,6 @@ A `Transform` stream finished with data still in the write buffer.
 
 The initialization of a TTY failed due to a system error.
 
-<a id="ERR_TTY_WRITABLE_NOT_READABLE"></a>
```

Comment:
Maybe we should move this also to `Legacy Node.js Error Codes` section?

Reply:
@vsemozhetbyt Thanks, I did not know about that :) Done!

Reply:
This never shipped into a Release. it’s safe to remove.

Reply:
It seems we now have a section for such errors as well)

https://github.com/nodejs/node/blob/master/doc/api/errors.md#other-error-codes

Reply:
@vsemozhetbyt Oh, wow – thanks again :smile: Done!